### ISSUE TITLE
irb insert control characters when using del key on OSX

### ISSUE THREAD

**Issue by KervyN** [rbenv/rbenv#1466]
Hi,
I am not sure if this is the correct repo to create this issue.

### Build env:
- OSX Ventura M1
- Installation steps followed with `brew`

### Versions that I encountered with this problem:
- 3.1.2
- 3.1.3

### Problem:
Using `del` `home` `end` keys in `irb` leads to control character output: `^[[H^[[F^[[3~`

### Expectation:
Jump to beginning/end of line, delete character behind cursor.

### What I tried:
- Figure it out on my own with google and brain
- fresh reinstallation (removing `readlines`, `rbenv` and `ruby-build` from brew. Removing all ruby version from rbenv. Start fresh)
- Change terminal (iterm2 -> terminal)

### Why I think this might be a bug in rbenv/ruby-build:
- preinstalled OSX ruby works fine in all constellations
- Problem does not exists in SSH sessions on a linux host, where I use rbenv
- Basically never encountered it somewhere else

**Comment by KervyN** [comment]
Seems to be an IRB issue: https://github.com/ruby/irb/issues/330

**Comment by mislav** [comment]
Thanks for the pointer. Closing as not an issue with ruby-build.
